pasqal_cloud/_version.py

```diff
@@ -13,4 +13,4 @@
 # limitations under the License.
 
 
-__version__ = "0.11.1"
+__version__ = "0.11.2"
```

Comment:
As I just released it, rn it would be 0.11.3 instead but there is https://github.com/pasqal-io/pasqal-cloud/pull/131 open so if they merge it before you, you will have to update it to 0.11.4 (don't forget to update the changelog version as well if it happens)